### DISCUSSION TITLE
Question - set bodyAttribute of column based on the row value

### DISCUSSION THREAD

**Discussion by dr3nyl** [Power-Components/livewire-powergrid#771] [Q&A] [answered]
I'm trying to set text/bg color of a row depending on its value. Example: status is active(green) and inactive(red)
I find `bodyAttribute`  but not sure how to insert a condition based on the value.

**Comment by younus93** [comment]
Were you able to make progress on this? Would have been nicer if we could pass a closure and return the necessary styling.

**Comment by dansysanalyst** [comment, marked as answer]
You can use[ Action Rules](https://livewire-powergrid.com/table/action-rules.html#action-rules),  it might look something like this: 

```php
final class MyTable extends PowerGridComponent
{
//...
 public function actionRules(): array
    {
        return [
            Rule::rows()
                ->when(fn($user) => (bool) $user->is_active === false)
                ->setAttribute('class', 'bg-yellow-50 hover:bg-red-100'),

            Rule::rows()
                ->when(fn($user) => (bool) $user->is_active === true)
                ->setAttribute('class', 'bg-yellow-50 hover:bg-green-100')
        ];
    }
```

Also, you may check the classes in the default themes at your directory: `vendor/power-components/livewire-powergrid/src/Themes/`, more info on [custom themes](https://livewire-powergrid.com/table/custom-theme.html) in the doc.

**Reply by younus93** [reply]
Thanks. Will check it out.
